packages/lb-components/src/views/MainView/sidebar/PointCloudToolSidebar/components/rectRotateSensitivitySlider/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
New components need to add file comments
<img width="678" alt="image" src="https://github.com/open-mmlab/labelbee/assets/25564864/1c2d87e6-0640-4cd0-9498-8b7e2d95188d">